subsys/emds/emds_new.c

```diff
@@ -141,23 +142,21 @@ static int emds_entries_size(uint32_t *size, size_t alignment)
 		entries++;
 	}
 
-	*size = DIV_ROUND_UP(*size, alignment) * alignment;
-	*size += DIV_ROUND_UP(sizeof(struct emds_snapshot_metadata), alignment) * alignment;
+	*size = ROUND_UP(*size, alignment);
+	*size += ROUND_UP(sizeof(struct emds_snapshot_metadata), alignment);
 
 	return entries;
 }
 
-uint32_t emds_store_size_get(void)
+int emds_store_size_get(size_t *store_size)
 {
-	uint32_t store_size;
-
 	if (!emds_initialized) {
 		return -ECANCELED;
 	}
 
-	(void)emds_entries_size(&store_size, partition[0].fp->write_block_size);
+	emds_entries_size(store_size, partition[0].fp->write_block_size);
```

Comment:
I was unclear in the previous comment, as we need to handle the return value. 
If return value "entries=0" should we then set store_size to 0 as it can something else, and give a error value as return value from the function. 
If we do nothing with the return value, we need to set it to (void), or Coverity will give fault feedback.

Reply:
`store_size` is zero if nothing to store. https://github.com/nrfconnect/sdk-nrf/pull/23024/files#diff-1ec6a7acf5e72f23f1385d64843bbe68477aedcd608a75b2b2814ccca7a47b0aR131